File: examples/python/fs/fs.py
import os

class WitWorld:
    def read_file(self, filename: str) -> str:
        path = os.path.join("/data", filename)
        try:
            with open(path, "r") as f:
                return f.read()
        except Exception as e:
            return f"Error reading file: {e}"

File: examples/ts/fs/fs.ts
import * as fs from 'fs';
import * as path from 'path';

export function readFile(filename: string): string {
    const filePath = path.join('/data', filename);
    try {
        return fs.readFileSync(filePath, 'utf8');
    } catch (err: any) {
        return `Error reading file: ${err.message}`;
    }
}
